docs/docs/module_guides/observability/instrumentation.md

```diff
@@ -25,12 +25,59 @@ Listed below are the core classes as well as their brief description of the
 Using the new `instrumentation` module involves 3 high-level steps.
 
 1. Define a `dispatcher`
-2. Define and attach your `EventHandler`'s to `dispatcher`
-3. Define and attach your `SpanHandler` to `dispatcher`
+2. (Optional) Define and attach your `EventHandler`'s to `dispatcher`
```

Comment:
Mentioned in the notebook again, if you want you can mention that `NullEventHandler` are used by default, similarly for `SpanHandler`. Might not be necessary information though, so totally up to you -- i'm good either way.

Reply:
Yea I think thats more of an implementation detail, rather than a user-facing detail? But thanks for the reminder, definitely forgot that was a thing.